src/Yandex/Allure/Adapter/Allure.php

```diff
@@ -167,7 +167,7 @@ protected function saveToFile($testSuiteUuid, TestSuite $testSuite)
         if ($testSuite->size() > 0) {
             $xml = $testSuite->serialize();
             $fileName = $testSuiteUuid . '-testsuite.xml';
-            $filePath = Provider::getOutputDirectory() . DIRECTORY_SEPARATOR . $fileName;
+            $filePath = Provider::getOutputDirectory() . \DIRECTORY_SEPARATOR . $fileName;
```

Comment:
Is this backslash intentional? :)

Reply:
Yes, this has been done by CS-Fixer. We have `native_constant_invocation` rule enabled:
https://github.com/ezsystems/ezplatform-code-style/blob/master/src/lib/PhpCsFixer/Config.php#L140
(doc for the rule is here: https://github.com/FriendsOfPHP/PHP-CS-Fixer)